linkify_changelog.py

```diff
@@ -0,0 +1,14 @@
+import fileinput
```

Comment:
II don't  really like randomly throwing python into this repo, now it is a dependency required for contributing to this repo while it's a go repo. I don't think that's great developer experience. 

Seems trivial to rewrite this in go copy-pasting the regex.

```
package main

import (
	"bufio"
	"fmt"
	"log"
	"os"
	"regexp"
)

func main() {
	file, err := os.Open("./changelog.md")
	if err != nil {
		log.Fatal(err)
	}
	defer file.Close()

	m := regexp.MustCompile(`#([0-9]*)`)

	scanner := bufio.NewScanner(file)
	for scanner.Scan() {
		line := m.ReplaceAllString(scanner.Text(), `[#$1](https://github.com/livepeer/go-livepeer/pull/$1)`)
		fmt.Println(line)
	}

	if err := scanner.Err(); err != nil {
		log.Fatal(err)
	}
}
```

Input
```
#1 hello world (@kyriediculous)
#2 hello mars (@yondonfu)
```

Output
```
$ go run changelog-link.go
[#1](https://github.com/livepeer/go-livepeer/pull/1) hello world (@kyriediculous)
[#2](https://github.com/livepeer/go-livepeer/pull/2) hello mars (@yondonfu)
```

Reply:
Good call on sticking with Go for this script to limit the number of language dependencies for the repo. Converted the script to Go in 876d96f and updated the linkify command in `doc/releases.md` 5d41439.

> now it is a dependency required for contributing to this repo while it's a go repo. I don't think that's great developer experience.

Just to add a bit of clarification to this point - most contributors wouldn't be expected to have to run this script and the only person that would be expected to run this script (once when a release is being created) is the release coordinator.